From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@pcgeek86__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

pcgeek86 commented:
Closing pull request, as additional clarification was added to the Yammer thread.

v-nagta committed:
USER STORY 1024674 SaaS App Tutorial: MOVEit Transfer Tutorial Update

v-nagta committed:
Updated

mumian committed:
update wasbs to wasb

mumian committed:
add include file

mumian committed:
add a new article on secure transfer

v-nagta committed:
updated

mumian committed:
update links

mumian committed:
simplify the cluster creation section

mumian committed:
add include file to the articles

mumian committed:
update links

mumian committed:
fix link

mumian committed:
add a screenshot

mumian committed:
update folder name

mumian committed:
format

mumian committed:
format

mumian committed:
update screenshot

mumian committed:
modify toc.md

venkatasiriki committed:
USER STORY 1033904 SaaS App Tutorial: Veracode Tutorial Update

venkatasiriki committed:
USER STORY 1033904 SaaS App Tutorial: Veracode Tutorial Update

v-nagta committed:
updated

v-nagta committed:
updated

venkatasiriki committed:
USER STORY 1033904 SaaS App Tutorial: Veracode Tutorial Update

v-nagta committed:
USER STORY 1033892 SaaS App Tutorial: Qlik Sense Enterprise Tutorial Update

v-nagta committed:
Merge branch 'master' into qliksense

MGoedtel committed:
new article and refactored deploy article

MGoedtel committed:
corrected warning items

MGoedtel committed:
Updated HRW article

MGoedtel committed:
Updated link reference

MGoedtel committed:
updated TOC

MGoedtel committed:
redid running runbooks article

venkatasiriki committed:
USER STORY 1033904 SaaS App Tutorial: Veracode Tutorial Update

v-nagta committed:
updated

v-nagta committed:
updated

venkatasiriki committed:
USER STORY 1025728 SaaS App Tutorial: Nomadic Tutorial Update

v-vikum2 committed:
USER STORY 1024865 SaaS App Tutorial: Infor Retail – Information Management Update

v-vikum2 committed:
updated

v-viinde committed:
USER STORY 1025732 SaaS App Tutorial: OfficeSpace Software Tutorial Update

venkatasiriki committed:
USER STORY 1033884 SaaS App Tutorial: Fuse Tutorial Update

venkatasiriki committed:
USER STORY 1033884 SaaS App Tutorial: Fuse Tutorial Update

v-viinde committed:
Updated

MGoedtel committed:
updated articles

MGoedtel committed:
updated warning and removed indexing of retired article

MGoedtel committed:
corrected warnings and improved readabiility

MGoedtel committed:
corrected broken link

MGoedtel committed:
updated h2 and link to article

MGoedtel committed:
corrected warning msg

MGoedtel committed:
corrected per warning msg

MGoedtel committed:
Updated parameter for POSH cmdlet

Commit:
Add SQL metrics

Commit:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

Commit:
refresh sql query

paulathurman committed:
edit pass: application-insights-custom-operations-tracking

v-nagta committed:
Merge branch 'master' into moveittransfer

v-nagta committed:
updated

v-vikum2 committed:
USER STORY 1024850 SaaS App Tutorial: EasyTerritory Update

v-vikum2 committed:
updated

v-vikum2 committed:
updated

mumian committed:
add spark steps:

mumian committed:
minor edit

mumian committed:
conflict

mumian committed:
format

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-insights-custom-operations-tracking

mumian committed:
edit

Commit:
mimi comments

Commit:
add to toc

Commit:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

mumian committed:
update hdinsight-python.md

paulathurman committed:
edit pass: application-insights-custom-operations-tracking

paulathurman committed:
edit pass: application-insights-custom-operations-tracking

Commit:
improve score

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-insights-custom-operations-tracking

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-insights-custom-operations-tracking

paulathurman committed:
edit pass: application-insights-custom-operations-tracking

MGoedtel committed:
corrected items per feedback

seanmck committed:
Filling in details on usage of parameters files

seanmck committed:
Remove unnecessary newline

seanmck committed:
Fixing json

seanmck committed:
Fixing relative link to ARM article

seanmck committed:
Fix mount path

seanmck committed:
Fixing tagging on CLI snippets

seanmck committed:
Fixing up command to use variable

MGoedtel committed:
updated redirect master file

MGoedtel committed:
corrected redirect file format issue

MGoedtel committed:
corrected error

mumian committed:
acrolinx

MGoedtel committed:
corected syntax error

MGoedtel committed:
corrected formatting error with json file

MGoedtel committed:
removed entry

MGoedtel committed:
added redirect

MGoedtel committed:
removed file referenced in redirection

MGoedtel committed:
corrected error

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-insights-custom-operations-tracking

seanmck committed:
Fixing merge

seanmck committed:
Fixing merge (take 2)

paulathurman committed:
edit pass: application-insights-custom-operations-tracking

cherylmc committed:
update

v-shils committed:
Merge pull request #15993 from v-nagta/moveittransfer

USER STORY 1024674 SaaS App Tutorial: MOVEit Transfer Tutorial Update

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into routing

v-shils committed:
Merge pull request #17550 from venkatasiriki/veracode

USER STORY 1033904 SaaS App Tutorial: Veracode Tutorial Update

MicrosoftGuyJFlo committed:
SaaS Tutorial Landing Page Updates

v-thepet committed:
Merge pull request #18205 from seanmck/aci-azurefiles-howto

Filling in more details on usage of parameters file for mounting Azure files volumes

neilpeterson committed:
updated image name and output

v-shils committed:
Merge pull request #17951 from v-nagta/qliksense

USER STORY 1033892 SaaS App Tutorial: Qlik Sense Enterprise Tutorial …

Commit:
Merge pull request #18231 from neilpeterson/aci-multi-update

updated image name and output

cherylmc committed:
update format

v-shils committed:
Merge pull request #18014 from venkatasiriki/nomadic

USER STORY 1025728 SaaS App Tutorial: Nomadic Tutorial Update

Commit:
Merge pull request #18230 from cherylmc/routing

update format

v-thepet committed:
Merge pull request #18068 from arramac/master

Add SQL query metrics article

Commit:
updates to web app naming

v-shils committed:
Merge pull request #18015 from v-vikum2/inforretailinformationmanagement

USER STORY 1024865 SaaS App Tutorial: Infor Retail – Information Management Update

v-shils committed:
Merge pull request #18016 from v-viinde/officespace

USER STORY 1025732 SaaS App Tutorial: OfficeSpace Software Tutorial U…

Commit:
Merge pull request #18233 from georgewallace/appgatewaywebapp2

app gateway : updates to web app naming

v-shils committed:
Merge pull request #18018 from venkatasiriki/fuse

USER STORY 1033884 SaaS App Tutorial: Fuse Tutorial Update

v-thepet committed:
Merge pull request #18232 from MicrosoftGuyJFlo/SaaSTutotialListUpdate

SaaS Tutorial Landing Page Updates

MGoedtel committed:
Update script per community feedback

v-shils committed:
Merge pull request #18105 from v-vikum2/easyterritory

USER STORY 1024850 SaaS App Tutorial: EasyTerritory Update

ShawnJackson committed:
Merge pull request #18148 from paulathurman/application-insights-custom-operations-tracking

edit pass: Application insights custom operations tracking

MGoedtel committed:
corrected issue with redirect file

Commit:
Merge pull request #18237 from MGoedtel/UpdateCRunAs7272017

Update script per community feedback

v-thepet committed:
Merge pull request #17958 from MGoedtel/UpdateHRW7242017

new article and refactored deploy article

Lisaco88 committed:
Merge pull request #17180 from mumian/securetransfer0712

update wasbs to wasb

barbkess committed:
change author

andretms committed:
Fixing hyperlinks for iOS and JavaScript SPA app

Changing hyperlinks for iOS and JavaScript SPA app from GitHub to new existing Guided Setup docs

andretms committed:
Fixing typo

Commit:
Merge pull request #18244 from barbkess/fix

change author

Ja-Dunn committed:
Merge pull request #18243 from andretms/patch-2

Fixing hyperlinks for iOS and JavaScript SPA app

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #18255 from MicrosoftDocs/FromPublicRepo

From public repo
